packages/next-app/src/contexts/StableSwapContext.tsx

```diff
@@ -20,19 +20,30 @@ export interface AnalysisData {
 
 interface StableSwapContextType {
   initialData: AnalysisData;
+  newData?: AnalysisData;
```

Comment:
IMO this distinction between initial/new data is a bit confusing. I asked ChatGPT for some ideas and here's a few:

> Original vs Updated: This terminology clearly signifies that the first set of parameters was the initial or original one, and the second set represents a change or an update.
> 
> Before vs After: This pair emphasizes the temporal order of the parameter sets, making it clear that the "Before" set was entered first, and the "After" set represents a change.
> 
> Primary vs Secondary: This pair avoids connotations of time or change, instead simply differentiating between the two sets of parameters. It still implies an order, with the Primary set being more important or entered first.
> 
> Start vs End: Similar to "Before vs After," but uses a different metaphor, implying a process or journey from the Start parameters to the End ones.
> 
> Input vs Comparison: This would imply that the "Input" parameters are the ones that the user is mainly interested in, and the "Comparison" parameters are being used to check, verify, or contrast against them.
> 
> Baseline vs Variant: This could imply that the Baseline parameters are the standard or default ones, and the Variant parameters represent a change or deviation from that standard.
> 
> Reference vs Test: This pair implies that the "Reference" parameters are the known, trusted, or verified ones, and the "Test" parameters are being evaluated against them.
> 
> Control vs Experimental: Borrowed from experimental sciences, this pair of terms could be useful if the app is often used for testing changes or innovations.
> 

I think the three last ones make more sense to our case, but wanted to paste the full output so that you maybe have any other ideas!

Reply:
Baseline and Variant sounds good